### PR TITLE
fix(tracing): fix unknown service name

### PR DESCRIPTION
**Pull request by galta95**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |

### CODE REVIEW

**Review by CptSchnitz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The following code shows a simple example of how to work with tracing. please no
 ```typescript
 import { Tracing } from '@map-colonies/telemetry';
 
-const tracing = new Tracing('my-tracer-name');
+const tracing = new Tracing();
```
Comment (syncush) [inline README.md:12]: add example with resource to match the function with Resource param

File: src/tracing/tracing.ts [hunk 2]
```diff
@@ -9,20 +9,24 @@ import { getTracingConfig, TracingConfig } from './config';
...
   public start(): void {
-    if (!this.config.isEnabled) {
+    const { version, isEnabled, serviceName, ...exporterConfig } = this.config;
+
+    if (!isEnabled) {
       return;
     }
 
-    const { version, isEnabled, serviceName, ...exporterConfig } = this.config;
+    let actualResource = new Resource({ 'service.version': version, 'service.name': serviceName });
```
Comment (syncush) [inline src/tracing/tracing.ts:23]: rename to resource

File: src/tracing/tracing.ts [hunk 3]
```diff
@@ -9,20 +9,24 @@ import { getTracingConfig, TracingConfig } from './config';
 export class Tracing implements TelemetryBase<void> {
   private provider?: NodeTracerProvider;
   private readonly config: TracingConfig;
-  public constructor(private readonly insturmentations?: InstrumentationOption[], private readonly serviceName?: string) {
+  public constructor(private readonly insturmentations?: InstrumentationOption[], private readonly resource?: Resource) {
```
Comment (syncush) [inline src/tracing/tracing.ts:12]: rename maybe to `userResource`?

**Review by syncush — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (galta95) [reply README.md:13]: done

File: src/tracing/tracing.ts (hunk 2 above)
Comment (galta95) [reply src/tracing/tracing.ts]: done

File: src/tracing/tracing.ts (hunk 3 above)
Comment (galta95) [reply src/tracing/tracing.ts]: done

**Review by syncush — APPROVED** [review]

**Pull request merged**